.github/workflows/check-source-branch.yml

```
@@ -0,0 +1,23 @@
+      - stable
+    types: [opened, synchronize, reopened]
+
+env:
+  ALLOWED_BRANCH: "next"
+
+jobs:
+  check-source:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check source branch
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)